content/en/blog/_posts/2022-01-14-SIG-Multicluster-Spotlight/index.md

```diff
@@ -0,0 +1,64 @@
+# Spotlight on SIG Multicluster
+
```

Comment:
Please remove these lines. The `---` that starts the front patter should be the first line.